### ISSUE TITLE
Finish Readme

### ISSUE THREAD

**Issue by DopeforHope** [DopeforHope/isisDataFetcher#1]

**Comment by Xuij** [comment]
Yes. Going to write a code in curl to access isis and fetch the data
-What other tools do we want to implement?

**Comment by DopeforHope** [comment]
Exactly, this should be the next step. But we should also keep in mind that  we have to catch zip files too and also we have to create things like a GUI, optimize code(for example make the week title array as String instead of float) and so on.

Also an intresting approach would be scrapy.
But dont worry I will create some issues soon.

**Comment by Xuij** [comment]
Since you're the owner, suggest you make a wiki of what tool you want to build exactly. It's important that we align our view and skills with what we work on. A basic structure will also help sort ideas and features in the future.

**Comment by Xuij** [comment]
Tips for installing the python version in the readme
